plugins/deals.py

```diff
@@ -0,0 +1,32 @@
+from cloudbot import hook
+from cloudbot.util import web
+import feedparser
+
+@hook.command('meh', autohelp=False)
+def meh():
+	'''List the current meh.com deal.'''
```

Comment:
Docstrings should use triple quotes, not triple single-quotes.
Command docstrings should also begin with `- ` if they do not specify any arguments, due to how the core generates the help response